pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic
+from sympy.physics.mechanics import ReferenceFrame, Particle, RigidBody, dynamicsymbols
+
+# local
+from ..bodies import Body, Ground
+from ..force import Force
+
+class TestBody():
+    def setup(self):
+        self.body = Body('Body')
+
+    def test_body_init(self):
+        # by default a rigidbody is created.
+        assert self.body.name == 'Body'
+        assert self.body.parent is None
+        assert self.body.child is None
+        assert isinstance(self.body.masscenter, Point)
+        assert isinstance(self.body.frame,    ReferenceFrame)
+        assert isinstance(self.body.mass, Symbol)
+        assert len(self.body.inertia) == 2
+        assert isinstance(self.body.inertia[0], Dyadic)
+        assert isinstance(self.body.inertia[1], Point)
+        assert isinstance(self.body.body, RigidBody)
+
+    def test_body_properties_value(self):
+        name = self.body.name
+        assert self.body.masscenter == Point(name + ' MassCenter')
+        assert self.body.frame == ReferenceFrame(name + ' ReferenceFrame')
+        assert self.body.mass == Symbol(name + ' Mass')
+
+    def test_body_particle(self):
+        point = Point('point')
+        mass = Symbol('mass')
+        self.particle_body = Body(masscenter=point, mass=mass)
+        assert isinstance(self.particle_body.body, Particle)
+        assert self.particle_body.masscenter == point
+        assert self.particle_body.mass == mass
+
+    def test_body_particle_add_force(self):
```

Comment:
It may be useful for the user to add a force easily to the particle point or the mass center of a rigidbody, but in general you need to be able to apply forces to arbitrary points.